pkg/oidcclient/login.go

```diff
@@ -216,6 +217,14 @@ func WithSessionCache(cache SessionCache) Option {
 	}
 }
 
+// WithOutWriter sets stderr io.Writer.
+func WithOutWriter(out io.Writer) Option {
```

Comment:
Sadly, this package is not under `internal` so this package is in use by other projects as a go package library. We should try to avoid breaking those other projects as much as possible (unless we feel that need to).

I missed this on my first review, but just noticed that `h.out` seems to have no default value in the `Login` function. I think it should be assigned `os.Stderr` in the new struct by default before the `opts` get run, right? Otherwise `WithOutWriter` isn't optional.

Reply:
Actually, does this need to be part of the public API? I can't think of any use case for when it should be a different output writer other than stderr.

Reply:
I made it "injectable" only for testing purposes. I wasn't at all thinking about this as part of the public API. Let's discuss next week.

Reply:
If its needed (or just preferred) for testing purposes then perhaps make the `WithOutWriter` function package-private and make sure that `h.out` gets a default value when it is not called?

Reply:
We decided to move `WithOutWriter` to be a test-only func